packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -828,7 +868,10 @@ class _DropdownButtonState<T> extends State<DropdownButton<T>> with WidgetsBindi
     return null;
   }
 
-  bool get _enabled => widget.items != null && widget.items.isNotEmpty && widget.onChanged != null;
+  bool get _enabled =>
```

Comment:
Use arrow function for single line only: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods